modules/fastcdr/2.1.3/patches/add_cmake_configure_file.patch

```diff
@@ -0,0 +1,338 @@
+diff --git bazel/BUILD.bazel bazel/BUILD.bazel
+new file mode 100644
+index 0000000..e5d9278
+--- /dev/null
++++ bazel/BUILD.bazel
+@@ -0,0 +1,5 @@
++py_binary(
++    name = "cmake_configure_file",
++    srcs = ["cmake_configure_file.py"],
++    visibility = ["//visibility:public"],
++)
+\ No newline at end of file
+diff --git bazel/cmake_configure_file.bzl bazel/cmake_configure_file.bzl
+new file mode 100644
+index 0000000..cbfb7de
+--- /dev/null
++++ bazel/cmake_configure_file.bzl
+@@ -0,0 +1,86 @@
++# -*- python -*-
++
++# Copied from the Drake project:
++# https://github.com/RobotLocomotion/drake/blob/17423f8fb6f292b4af0b4cf3c6c0f157273af501/tools/workspace/cmake_configure_file.bzl
++
++# Defines the implementation actions to cmake_configure_file.
++def _cmake_configure_file_impl(ctx):
++    arguments = [
++        "--input",
++        ctx.file.src.path,
++        "--output",
++        ctx.outputs.out.path,
++    ]
++    for item in ctx.attr.defines:
++        arguments += ["-D" + item]
++    for item in ctx.attr.undefines:
++        arguments += ["-U" + item]
++    for item in ctx.files.cmakelists:
++        arguments += ["--cmakelists", item.path]
++    ctx.actions.run(
++        inputs = [ctx.file.src] + ctx.files.cmakelists,
++        outputs = [ctx.outputs.out],
++        arguments = arguments,
++        env = ctx.attr.env,
++        executable = ctx.executable.cmake_configure_file_py,
++    )
++    return []
++
++# Defines the rule to cmake_configure_file.
++_cmake_configure_file_gen = rule(
++    attrs = {
++        "src": attr.label(
++            allow_single_file = True,
++            mandatory = True,
++        ),
++        "out": attr.output(mandatory = True),
++        "defines": attr.string_list(),
++        "undefines": attr.string_list(),
++        "cmakelists": attr.label_list(allow_files = True),
++        "cmake_configure_file_py": attr.label(
++            cfg = "host",
```

Comment:
This configuration is deprecated and should be replaced with `exec`.

Reply:
change `host` to `exec` at https://github.com/bazelbuild/bazel-central-registry/pull/1345/commits/e6ff09dbc45e3c63f2ffd1df3b0f60cc3af130c3